### ISSUE TITLE
exploding assembly3?

### ISSUE THREAD

**Issue by mandd3** [realthunder/FreeCAD_assembly3#196]
Hi,
any advice on achieving Exploded_Assembly with assembly3?

Thanks!

**Comment by realthunder** [comment]
I will add this feature soon. For now, I add a simple command to make it easy to do it manually. You will need to sync my LinkStage3 branch to get it. If you are not building FC yourself, please wait till next release.

Note that the LinkGroup created in the screenshot blew has synchronized placement to the source assembly. It creates a `Link` for each part inside assembly with `LinkTransform` set to `True`, which allows the user to apply additional transformation. The `TechDraw` workbench accepts `LinkGroup`, so you can easily make explode drawing with it.

![manual-explode](https://user-images.githubusercontent.com/1207888/52828768-10534180-3105-11e9-9a8f-5bcd27ec107d.gif)

**Comment by ceremcem** [comment]
A-ha! Isn't it opening the doors for ["Custom scenes"](https://github.com/realthunder/FreeCAD_assembly3/issues/186)?

**Comment by realthunder** [comment]
Exactly. I am think of putting this type of feature under a new sub group of the assembly container.

**Comment by dulouie** [comment]
That's real App:Link performance!  really neat

**Comment by mandd3** [comment]
Will be using assembly3 from now on.
thank you realthunder. Great work.
